plenum/common/messages/fields.py

```diff
@@ -120,10 +120,13 @@ def _specific_validation(self, val):
             return '{} is longer than {} symbols'.format(val, self._max_length)
 
 
-class SignatureField(FieldBase):
+class SignatureField(LimitedLengthStringField):
     _base_types = (str, type(None))
     # TODO do nothing because EmptySignature should be raised somehow
 
+    def __init__(self, max_length: int, **kwargs):
+        super().__init__(max_length=max_length, **kwargs)
```

Comment:
Do we need that __init__() really? And for other the same cases...

Reply:
removed